hack/gen-bundle/update-bundle.sh

```diff
@@ -43,6 +43,19 @@ yq write -i ${CSV} metadata.annotations.containerImage ${OPERATOR_IMAGE_DIGEST}
 TIMESTAMP=$(echo ${OPERATOR_IMAGE_INSPECT} | jq -r .[0].Created)
 yq write -i ${CSV} metadata.annotations.createdAt ${TIMESTAMP}
 
+# Add the features annotations per https://docs.openshift.com/container-platform/4.14/operators/operator_sdk/osdk-generating-csvs.html
+FEATURES=metadata.annotations.features.operators.openshift.io
+yq write -i ${CSV} ${FEATURES}/disconnected "false"
+yq write -i ${CSV} ${FEATURES}/fips-compliant "false"
+yq write -i ${CSV} ${FEATURES}/proxy-aware "false"
+yq write -i ${CSV} ${FEATURES}/tls-profiles "false"
+yq write -i ${CSV} ${FEATURES}/token-auth-aws "false"
+yq write -i ${CSV} ${FEATURES}/token-auth-azure "false"
+yq write -i ${CSV} ${FEATURES}/token-auth-gcp "false"
+yq write -i ${CSV} ${FEATURES}/cnf "false"
+yq write -i ${CSV} ${FEATURES}/cni "true"
+yq write -i ${CSV} ${FEATURES}/csi "false"
```

Comment:
I was looking at the link provided and I thought we did provide a csi plugin. We do have the `csi-node-driver` daemonset.

Reply:
Good point.  @matthewdupre thoughts?

My thoughts on this are that while we _include_ a CSI plugin in our install, we don't _provide_ a CSI plugin with our install.

My worry would be that people would assume that they don't need to install another CSI plugin on their cluster because Calico takes care of that, which isn't quite true.

Reply:
We don't provide CSI functionality to the end user. We use CSI for a very specific purpose within our produce to make one of our features (ALP) work. 

We're aren't a general purpose storage driver, so I suspect we shouldn't indicate that we are in this manifest.

Reply:
@tmjd are you happy to approve this now?